internal/npm_install/npm_install.bzl

```diff
@@ -333,7 +333,7 @@ cd /D "{root}" && "{npm}" {npm_args}
     # removeNPMAbsolutePaths is run on node_modules after npm install as the package.json files
     # generated by npm are non-deterministic. They contain absolute install paths and other private
     # information fields starting with "_". removeNPMAbsolutePaths removes all fields starting with "_".
-    print([node, repository_ctx.path(remove_npm_absolute_paths), root + "/node_modules"])
+    # print([node, repository_ctx.path(remove_npm_absolute_paths), root + "/node_modules"])
```

Comment:
Perhaps put this behind `repository_ctx.attr.quiet`:
```
    if not repository_ctx.attr.quiet:
        print([node, repository_ctx.path(remove_npm_absolute_paths), root + "/node_modules"])
```
That way if a user requests more debug logging with `quiet = False` on the `*_install` rule this will be included

Reply:
Yeah, that's a better idea.  Updated.